random-fu/random-fu.cabal

```diff
@@ -91,6 +91,7 @@ Library
   
   build-depends:        math-functions,
                         monad-loops >= 0.3.0.1,
+                        random >= 1.2.0,
```

Comment:
Should we have an upper bound here?

Reply:
> What is the overall plan here? Is this mainly for a 0.3.0.11 release, or is the plan to abandon the generalized `RandomSource` instances from #45, in particular [ff0ce7f](https://github.com/haskell-numerics/random-fu/commit/ff0ce7fd73cd48473aa095516a2a68c01ca21ba3) (which was reverted in #55, and reinstated in #56)?

My intention is to get everything back to working and yes abandon the generalized instances until we can find a way of adding them that doesn't cause conflicts. I have no idea how difficult this is.

My personal preference for improving these packages would be to see if we could get rid of randomsource and replace it by the new random package. I'd hope we could get an ever bigger performance improvement as well as simplying the code base.

I'd be interested in your thoughts (or indeed anyone's)

Reply:
I guess so - I'll put `>=1.2 && <1.3`